### DISCUSSION TITLE
GitHub Student Pack not accepting my school ID

### DISCUSSION THREAD

**Discussion by ameoverflow** [community/community#40395] [GitHub Education]
My school ID has personal information (such as name of my school, my full name) on one side, and valid until date on the other side, in form of little stickers that are put on every year to renew my school ID.
Therefore, as GitHub doesn't accept 2 photos, I can't supply enough proof to let me through.
![obraz](https://user-images.githubusercontent.com/31414969/204669679-e0efc402-355d-4e1b-82c2-f2decaac43bc.png)
(also "You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.": I would like not to do that because of privacy)
Edit: tried adding my school email as a second email on my account, and submitting class timetable as a proof. Currently waiting for response
Edit 2: my request has been approved

**Comment by SilasRodrigues19** [comment]
@nolemretaWxd Hi.

There is a similar discussion at this link: #37159

**Reply by ameoverflow** [reply]
No, it isn't. That discussion is about not being able to verify because OPs school is in partner program. Mine isn't, and I'm unable to verify because I have no way of providing required information (school name and valid until date) from my school ID because I can submit only one photo
